docs/modules/ROOT/pages/release-notes/release-notes-32-0.adoc

```diff
@@ -0,0 +1,78 @@
+# Release Notes - Payara Platform Enterprise 5.32.0
+
+### Supported APIs and Applications
+* Jakarta EE 8
+* Java EE 8 Applications
+* MicroProfile 4.1
+
+
+#### Breaking Changes
+
+##### Hazelcast 4 Upgrade
+
+Since this Enterprise version contains an upgraded version of Hazelcast, there is no rolling upgrade possible due to the changes within Hazelcast. A domain or set of Payara Micro instances having a mixture of Hazelcast 3.x and Hazelcast 4.x results in errors.
+
+Since the Cluster Password is no longer supported on Hazelcast, the asadmin command `set-hazelcast-configuration` no longer has the optional `--clusterPassword` option.
+
+##### Payara Nexus Repository Changes
+
+The structure of our Payara Nexus repository has changed. The name change, however, means that you might need to change the URLs pointing to the repository in your repository manager, CI environment, or development environment if you make use of the `payara-enterprise` repository.  You can read more details about the change and the new structure in our knowledge base article; https://support.payara.fish/hc/en-gb/articles/4408617212177[Payara Nexus repository changes after October 18th 2021].
```

Comment:
The structure of nexus repositories hasn't changed, it's the structure of the Nexus server that changed. Individual repositories retain the same structure, but the name of some of them is changed.
```suggestion
The structure of our Payara Nexus server has changed. This means that you might need to change the URLs pointing to the repository in your repository manager, CI environment, or development environment if you make use of the `payara-enterprise` repository.  You can read more details about the change and the new structure in our knowledge base article; https://support.payara.fish/hc/en-gb/articles/4408617212177[Payara Nexus repository changes after October 18th 2021].
```